linkerd/app/outbound/src/tcp/tests.rs

```diff
@@ -721,9 +725,9 @@ async fn profile_endpoint_propagates_conn_errors() {
     let mut server = build_server(cfg, profiles, resolver, connect);
 
     let svc = server.new_service(listen::Addrs::new(
-        ([127, 0, 0, 1], 4140).into(),
-        ([127, 0, 0, 1], 666).into(),
-        Some(ep1),
+        Local(ServerAddr(([127, 0, 0, 1], 4140).into())),
+        Remote(ClientAddr(([127, 0, 0, 1], 666).into())),
+        Some(OrigDstAddr(ep1)),
     ));
```

Comment:
nit/TIOLI: seems like the `addrs` helper function in the HTTP tests could be pulled out into the top-level test support module and we could use it here as well?

Reply:
yyyeah i like that idea but let's do that in a followup.